### ISSUE TITLE
Unicode error 

### ISSUE THREAD

**Issue by vanroekel** [MPAS-Dev/MPAS-Analysis#322]
Running the latest origin/develop on edison and anvil, I now get this error
```
/global/project/projectdirs/acme/software/anaconda_envs/edison_e3sm_unified_1.1.2_py2_nox/lib/python2.7/site-packages/matplotlib/font_manager.py:280: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.
  'Matplotlib is building the font cache using fc-list. '
Traceback (most recent call last):
  File "./run_mpas_analysis", line 544, in <module>
    analyses = build_analysis_list(config, refConfig)
  File "./run_mpas_analysis", line 64, in build_analysis_list
    from mpas_analysis import ocean
  File "/global/u2/l/lvroekel/MPAS-Analysis/mpas_analysis/ocean/__init__.py", line 12, in <module>
    from mpas_analysis.ocean.time_series_temperature_anomaly import \
  File "/global/u2/l/lvroekel/MPAS-Analysis/mpas_analysis/ocean/time_series_temperature_anomaly.py", line 81
SyntaxError: Non-ASCII character '\xce' in file /global/u2/l/lvroekel/MPAS-Analysis/mpas_analysis/ocean/time_series_temperature_anomaly.py on line 81, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```
These seem to be related to unicode characters in labels that are a part of python 3.  They appear to be imported from future in all the right places, but I still get this error.  The config file for this run is ```/global/homes/l/lvroekel/MPAS-Analysis/config.highRes```.  The steps I did to recreate this error are
```
git reset --hard origin/develop
module use /global/project/projectdirs/acme/software/modulefiles/all
module unload python
module load e3sm-unified
./run_mpas_analysis config.highRes
```

Am I missing a required step here?

**Comment by xylar** [comment]
@vanroekel, I'm nearly certain this is the same issue as #318 and is addressed by #319, which I plan to merge in 10 minutes or so.

**Comment by milenaveneziani** [comment]
yes, this looks like exactly the error I was getting.

**Comment by vanroekel** [comment]
I have confirmed this is addressed by #319
